Pocket/pocket-ios
# SwiftUI Home

## Goal
This PR contains a rewritten version of Home 100% SwiftUI

> [!NOTE]
> The new Home is still WIP and can only be seen in debug builds, in an additional tab called SwiftUI Home

## Test Steps
* Smoke test SwiftUI home and make sure you see the same behaviors of the existing Home

> [!NOTE]
> The following features are not yet available in SwiftUI Home
> - Native collections
> - Slate detail
> - Signed out home
> - Pull to refresh
> - Loading animation 
> - Haptics on actions (e.g. Save, Favorite)
